openapi/v2/core-metadata.yaml

```diff
@@ -552,6 +562,147 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/DeviceService'
+    ProvisionWatcher:
+      description: "A ProvisionWatcher defines the filtering criteria for device auto discovery."
+      type: object
+      properties:
+        apiVersion:
+          type: string
+        id:
+          type: string
+          format: uuid
+          description: Uniquely identifies this request. For implementation, recommend this value be generated by the type's constructor
```

Comment:
This make it sounds like this is the `requestID` and not the entity id. Please reword here and below